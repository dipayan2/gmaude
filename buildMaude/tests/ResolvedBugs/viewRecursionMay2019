#!/bin/sh

MAUDE_LIB=$srcdir/../../src/Main
export MAUDE_LIB

../../src/Main/maude \
  < $srcdir/viewRecursionMay2019.maude -no-banner -no-advise \
  > viewRecursionMay2019.out 2>&1

diff $srcdir/viewRecursionMay2019.expected viewRecursionMay2019.out > /dev/null 2>&1
