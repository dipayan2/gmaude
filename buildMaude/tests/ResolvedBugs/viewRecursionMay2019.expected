Warning: <standard input>, line 5 (view NatSet): recursive use of view NatSet
    is not allowed.
Warning: <standard input>, line 5 (view NatSet): unusable view NatSet.
Bye.
